xml/System.Diagnostics.Tracing/EventCounter.xml

```diff
@@ -20,7 +20,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides the ability to collect statistics for very frequent events through the  <see cref="T:System.Diagnostics.Tracing.EventSource" /> class.</summary>
     <remarks>To be added.</remarks>
```

Comment:
Is it your intent to leave the 'To be added' in the remarks.   Presumably we should either remove that text or add the real text (which I can help with).

Reply:
The "To be added." is the default string from our new reflection tooling and they disappear on the build content. So we're not touching those.

But if we want to add remarks, for sure. I'd love your help with that!